packages/build/src/plugins_core/functions/error.js

```diff
@@ -152,10 +152,10 @@ package = "@netlify/plugin-functions-install-core"
 // because `optionalDependencies` can make `import()` fail, but we don't want
 // to error then. However, if the `package.json` is invalid, we fail the build.
 const isPackageJsonError = function (error) {
-  return error.message.includes(PACKAGE_JSON_ORIGINAL_MESSAGE)
+  return PACKAGE_JSON_ORIGINAL_MESSAGES.some((msg) => error.message.includes(msg))
 }
 
-const PACKAGE_JSON_ORIGINAL_MESSAGE = 'is invalid JSON'
+const PACKAGE_JSON_ORIGINAL_MESSAGES = ['is invalid JSON', 'in JSON at position']
```

Comment:
Hmm `in JSON at position` feels a bit too broad, but I guess there's no better way to catch there errors?

Reply:
Yeah i tried to match as a much as possible, but there is not much. The whole error is `SyntaxError: Unexpected token { in JSON at position 1`